src/api/eth/send.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (efx, wrapperAbi, address, action, args, value = 0) => {
+  const { web3 } = efx
+
+  const contract = new web3.eth.Contract(wrapperAbi, address)
+
+  // REVIEW: Shall we use estimateGas here? maybe show the user what the gas
+  // will be?
+  // see: https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#methods-mymethod-estimategas
```

Comment:
@hems We can estimate gas but the gas of most calls (deposit/withdraw) to lock and unlock should be fixed so we might be happy to set these as fixed to avoid using estimateGas.

Reply:
@plutoegg i can see if we don't provide gas/gasPrice it does call the estimateGas automatically. 
I guess fixing those values would simply save us some time on each send request.